hyph-utf8/tex/generic/hyph-utf8/patterns/tex/hyph-cs.tex

```diff
@@ -1,33 +1,27 @@
-% This file has been converted for the hyph-utf8 project from czhyphen.tex and
-% czhyphen.ex, whose author has been identified as Pavel Ševeček <pavel at
-% lingea.cz>.  The licence terms are unchanged.
+% title: Czech hyphenation patterns
+% copyright: Copyright (c) Pavel Ševeček
+%     Lingea s.r.o., Brno, Czech Republic
+%     E-mail pavel@lingea.cz
+% notice: This file is part of the hyph-utf8 package.
+%     See http://www.hyphenation.org for details on the project.
+% language:
+%     name: Czech
+%     tag: cs
+% licence:
+%     name: GPL
+%     version: 2
+%     or_later: true
+%     url: http://www.gnu.org/licenses/gpl.html
+% hyphenmins:
+%     generation:
+%         left: 2
+%         right: 2
+%     typesetting:
+%         left: 2
+%         right: 2
+% changes:
+%     - v3, 1995
```

Comment:
We need an explicit version field, not just "changes".

Reply:
Replaced with `version` key.